Flexa-Agents/Large-deposits/src/agent.spec.ts

```diff
@@ -0,0 +1,253 @@
+import { Finding, FindingSeverity, FindingType, HandleTransaction, TransactionEvent } from "forta-agent";
+import {
+  createAddress,
+  TestTransactionEvent,
+  encodeParameters,
+  MockEthersProvider,
+} from "forta-agent-tools";
+import { provideHandleTransaction } from "./agent";
+import { BigNumber } from "ethers";
+import { Interface } from "@ethersproject/abi";
+import abi from "./abi";
+import { leftPad } from "web3-utils";
+
+const toBytes32 = (n: string) => leftPad(BigNumber.from(n).toHexString(), 64);
+const testAmp: string = createAddress("0xdef1");
+const testFlexa: string = createAddress("0xf1e4a");
+const testThreshold: BigNumber = BigNumber.from(100);
+const testAmpIFace: Interface = new Interface(abi.AMP_TOKEN);
+const testFlexaIFace: Interface = new Interface(abi.COLLATERAL_MANAGER);
+const testFlag: string =
+  "0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff";
+
+const createFinding = ([
+  value, fromPartition,
+  operator, from,
+  destinationPartition,
+  to, operatorData,
+]: string[]) => Finding.fromObject({
+  name: "Large Deposit",
+  description: "Large Deposit into staking pool",
+  alertId: "FLEXA-2",
+  severity: FindingSeverity.Info,
+  type: FindingType.Info,
+  metadata: {
+    value, fromPartition,
+    operator, from,
+    destinationPartition,
+    to, operatorData,
```

Comment:
For readability I'd recommend using the same formatting as you have in the `agent.ts` file where each part of the metadata has a key. See [here](https://github.com/NethermindEth/Forta-Agents/blob/548867fb5911110157e606f3c304d12bc99c9126/Flexa-Agents/Large-deposits/src/agent.ts#L31-L39).